package.json

```diff
@@ -1,23 +1,17 @@
 {
   "name": "@ensdomains/dnsregistrar",
   "main": "dist/dnsregistrar.js",
```

Comment:
Can you get rid of `dist/dnsregistrar.js` and replace it with something like https://github.com/ensdomains/ens/blob/master/index.js ?

Reply:
I've removed the main reference. There doesn't seem like there's much point in adding that given our pending change to a monorepo.